### ISSUE TITLE
Music skips randomly 

### ISSUE THREAD

**Issue by darker-than-black-69** [25huizengek1/ViTune#587]
While playing it suddenly skips to next song even if song hasn't ended yet.

**Comment by Wqttzicue** [comment]
Turn off auto-skip

**Comment by Monoxide8969** [comment]
> Does ‘auto-skip’ skip the song in the middle, randomly?

No, unless the playback encounters some kind of error. It happens to me too, even if less frequently than before. I don't know what it is, but something is definitely off

**Comment by Wqttzicue** [comment]
> > Turn off auto-skip
> 
> Does ‘auto-skip’ skip the song in the middle, randomly?

Yes, it can skip some part of song, as i understood.
And i had same issue, and that helped

**Comment by 25huizengek1** [comment]
@darker-than-black-69 Please provide some logs with this, along with a song I can reproduce this with. My bold guess is that YouTube sends bad chunks and the player just skips over them.

**Comment by UcraNerd** [comment]
I have auto-skip off and it does it regardless
